lib/phoenix/test/channel_test.ex

```diff
@@ -356,13 +394,36 @@ defmodule Phoenix.ChannelTest do
 
   In the assertion above, we don't particularly care about
   the data being sent, as long as something was sent.
+  """
+  defmacro assert_broadcast(event, payload) do
+    quote do
+      assert_receive %Phoenix.Socket.Broadcast{event: unquote(event),
+                                               payload: unquote(payload)}, 100
+    end
+  end
 
-  The timeout is in miliseconds and defaults to 100ms.
+  @doc """
+  Same as assert_broadcast/2 but allows you to customize the timeout.
   """
-  defmacro assert_broadcast(event, payload, timeout \\ 100) do
+  defmacro assert_broadcast(event, payload, timeout) do
     quote do
       assert_receive %Phoenix.Socket.Broadcast{event: unquote(event),
                                                payload: unquote(payload)}, unquote(timeout)
     end
   end
+
+  @doc """
+  Same as assert_broadcast/2 but adds the option to specify the topic of the broadcast.
+
+      assert_broadcast "room:123", "some_event", %{"data" => _}
+
+  The default timeout is 100ms, but can be customized.
+  """
+  defmacro assert_broadcast(topic, event, payload, timeout \\ 100) do
```

Comment:
Oops, looks like this clause won't match. Is there a better way to handle these functions with the default timeout? There is probably something simpler than what I'm trying to do